src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -161,6 +161,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IPublicationRepository, PublicationRepository>();
             services.AddSingleton<DataServiceMemoryCache<BoundaryLevel>, DataServiceMemoryCache<BoundaryLevel>>();
             services.AddSingleton<DataServiceMemoryCache<GeoJson>, DataServiceMemoryCache<GeoJson>>();
+            services.AddTransient<ITableStorageService, TableStorageService>(s =>
+                new TableStorageService(
+                    publicStorageConnectionString,
+                    new StorageInstanceCreationUtil()));
```

Comment:
Just wondering what's this required for as I wouldn't have expected Data API to require anything from Azure table storage.

Reply:
No you're right - it's gone now thanks.  Only barely clinging on anymore in Admin and the Publisher...